config.yaml

```diff
@@ -373,3 +373,8 @@ target:
       path: target-inputs/ensembl
       jq: '.genes[] | {id: .id, biotype: .biotype, description: .description, end: .end, start: .start, strand: .strand, chromosome: .seq_region_name, approvedSymbol: .name, transcripts: .transcripts, signalP: .SignalP, uniprot_trembl: ."Uniprot/SPTREMBL", uniprot_swissprot: ."Uniprot/SWISSPROT"}'
   miniset: yes
+literature:
```

Comment:
We also need the raw files provided by EPMC from `gs://otar025-epmc`

Reply:
I mean at the ETL stage, we don't copy that information into our pre-data-releases bucket